src/app/shared/state/note/notes.reducer.ts

```diff
@@ -31,20 +31,25 @@ const converter = new showdown.Converter({
 export default function reducer(state = initialState, action: ActionWithPayload<any>) {
   switch (action.type) {
     case NotesActions.LOAD_NOTES: {
-      reduce(action.payload, function (result, note) {
-        note = cloneDeep(note);
-        note.content = converter.makeHtml(note.content)
-        return action.payload;
-      }, {});
+      const newPayload = action.payload.map((note: Note) => {
+        note.content = converter.makeHtml(note.content);
+        return note;
+      });
+      
       return {
-        entities: action.payload,
+        entities: newPayload,
         sortBy: state.sortBy ? state.sortBy : '+date'
       };
     }
 
     case NotesActions.LOAD_DB_NOTES: {
+      const newPayload = action.payload.entities.map((note: Note) => {
```

Comment:
no need for this. I separated load db notes action because the notes we store in indexedDB are already html.